code_search/kubeflow/components/submit-code-embeddings-job.libsonnet

```diff
@@ -30,10 +30,10 @@
 	              "python2",
 	              "-m",
 	              "code_search.dataflow.cli.create_function_embeddings",
-	              "--runner=DataflowRunner",	              
+	              "--runner=DataflowRunner",
 	              "--project=" + params.project,
-	              "--token_pairs_table=" + params.tokenPairsBQTable,
-	              "--function_embeddings_table=" + params.functionEmbeddingsBQTable,
+	              "--token_pairs_table=" + params.project + ":" + params.bqDataset + ".token_pairs",
+	              "--function_embeddings_table=" + params.project + ":" + params.bqDataset + ".code_embeddings_" + params.bqSuffix,
```

Comment:
Why did you make this change?  Why hardcode the fact that the token_pairs and function_embeddings_table always have these patterns?

In general when choosing a pattern like this e.g. the names of the tables we should do this at the highest level; i.e. have parameters that compeletly specify the value e.g. tokenPairsBQTable or functionEmbeddingsBQTable

This way its very easy to override or change the location of the table; the user doesn't need to reverse engineer the code to figure out which parameters to set to affect things.

Reply:
make sense. done.